src/data/faq_de.json

```diff
@@ -2245,7 +2245,7 @@
                                     "<b>Handlungsempfehlungen:</b>",
-                                    "<ul><li>Lassen Sie sich testen. Bei einer Warnung über ein erhöhtes Risiko („rote Kachel“) haben Sie Anspruch auf einen kostenlosen Test. Das muss nicht notwendigerweise ein PCR-Test sein.</li><li>Empfohlen sind mindestens 7 Tage, besser 10 Tage, für die Zeitdauer der Selbstquarantäne ab dem Tag der letzten Risiko-Begegnung. Das Datum der letzten Begegnung mit erhöhtem Risiko finden Sie in der roten Kachel.</li><li>Verzichten Sie nach Möglichkeit auf Besuche und Aktivitäten, bei denen Sie mit anderen Menschen zusammenkommen. Lassen Sie sich Lebensmittel liefern oder bitten Sie jemanden, die Einkäufe zu erledigen; meiden Sie öffentliche Verkehrsmittel und verzichten Sie auf Flug- und Zugreisen.</li><li>Vermeiden Sie insbesondere Kontakte zu vulnerablen Personen und Gruppen, beispielsweise zu Älteren und Vorerkrankten.</li><li>Minimieren Sie auch Kontakte im eigenen Haushalt.</li><li>Wenn Ihr Arbeitgeber Ihnen die Möglichkeit gibt, arbeiten Sie von zu Hause aus.</li><li>Achten Sie mindestens für 10 Tage (ab Risiko-Begegnung) auf Krankheitssymptome und wenden Sie sich an Ihre Hausarztpraxis, falls Sie sich krank fühlen oder Symptome auftreten sollten.</li><li>Halten Sie die Hygieneregeln ein.</li></ul>",
+                                    "<ul><li>Lassen Sie sich testen.  Nach <a href='https://www.gesetze-im-internet.de/coronatestv_2021-10/__4a.html' target='_blank' rel='noopener'>§4a Testverordnung</a> haben Sie Anspruch auf einen kostenlosen Schnelltest. Weitere Infomrationen dazu finden Sie hier: <a href='https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html#c23926' target='_blank' rel='noopener'>Fragen und Antworten zu COVID-19 Tests (BMG)</a>. Das muss nicht notwendigerweise ein PCR-Test sein.</li><li>Empfohlen sind mindestens 7 Tage, besser 10 Tage, für die Zeitdauer der Selbstquarantäne ab dem Tag der letzten Risiko-Begegnung. Das Datum der letzten Begegnung mit erhöhtem Risiko finden Sie in der roten Kachel.</li><li>Verzichten Sie nach Möglichkeit auf Besuche und Aktivitäten, bei denen Sie mit anderen Menschen zusammenkommen. Lassen Sie sich Lebensmittel liefern oder bitten Sie jemanden, die Einkäufe zu erledigen; meiden Sie öffentliche Verkehrsmittel und verzichten Sie auf Flug- und Zugreisen.</li><li>Vermeiden Sie insbesondere Kontakte zu vulnerablen Personen und Gruppen, beispielsweise zu Älteren und Vorerkrankten.</li><li>Minimieren Sie auch Kontakte im eigenen Haushalt.</li><li>Wenn Ihr Arbeitgeber Ihnen die Möglichkeit gibt, arbeiten Sie von zu Hause aus.</li><li>Achten Sie mindestens für 10 Tage (ab Risiko-Begegnung) auf Krankheitssymptome und wenden Sie sich an Ihre Hausarztpraxis, falls Sie sich krank fühlen oder Symptome auftreten sollten.</li><li>Halten Sie die Hygieneregeln ein.</li></ul>",

```

Comment:
"Infomrationen" is a typo

Reply:
Typo has been fixed in [`070ebd2`](https://github.com/corona-warn-app/cwa-website/pull/2921/commits/070ebd26220fa47e63bc42bee9cd03452232076d). Thanks for spotting! Please re-review.